src/Data/Yaml.php

```diff
@@ -30,8 +30,8 @@ public static function encode($data): string
         // change to english numerics to avoid issues with floats
         setlocale(LC_NUMERIC, 'C');
```

Comment:
We can probably remove this locale stuff around the parse command. It was only added because Spyc is so buggy.

Reply:
Are you sure? Wasn’t that added over the whole comma vs. dot separator issues?

Reply:
Yes, but that should ideally be handled directly in the library. It was a bug in Spyc that Spyc didn't care about the locale. We have unit tests for this case and they should still pass with Symfony Yaml without the locale hack, otherwise Symfony Yaml is also buggy.